File: src/main/java/br/com/dusty/dkits/listener/mechanics/LeavesDecayListener.java
package br.com.dusty.dkits.listener.mechanics;

import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.block.LeavesDecayEvent;

public class LeavesDecayListener implements Listener {
	
	@EventHandler
	public static void onLeavesDecay(LeavesDecayEvent event){
		event.setCancelled(true);
	}
}

File: src/main/java/br/com/dusty/dkits/listener/mechanics/LeavesDecayListener.kt
package br.com.dusty.dkits.listener.mechanics

import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.block.LeavesDecayEvent

object LeavesDecayListener: Listener {

	@EventHandler
	fun onLeavesDecay(event: LeavesDecayEvent) {
		event.isCancelled = true
	}
}
